.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.1
```

Comment:
I'm not sure if this matters, but we may not want it in VC.

Reply:
my bad, I just removed it